odoo/addons/base/models/ir_mail_server.py

```diff
@@ -121,6 +121,14 @@ class IrMailServer(models.Model):
     smtp_debug = fields.Boolean(string='Debugging', help="If enabled, the full output of SMTP sessions will "
                                                          "be written to the server log at DEBUG level "
                                                          "(this is very verbose and may include confidential info!)")
+    max_email_size = fields.Float(string="Max Email Size", default=5,
+                                  help="Maximum email size that the outgoing server can send. "
+                                       "Don't set this value too high even if your server support high value "
+                                       "because it is also limited by what the recipient server can accept. "
+                                       "A good value is 5MB. "
+                                       "If a mail size exceeds that limit, its attachments will be turned into "
+                                       "downloadable links appended at the end of the email."
+                                  )
```

Comment:
Thanks for the improved help, it is updated. 
But I haven’t changed the string as in the form we have: 
Max Email Size	5.00	MB
(the unit is just after the field, instead of being in the label)